src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -79,6 +79,66 @@ public PersistentDataStore() {
     return users;
   }
 
+  /**
+   * Loads unique User object using UUID from the Datastore service and returns it.
+   *
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *     Datastore service
+   */
+  public User loadUser(UUID id) throws PersistentDataStoreException {
```

Comment:
Indention here should be 4 spaces, you can select all codes and use shift-command-f for autoformat